schemas/layer-4.cue

```diff
@@ -1,35 +1,45 @@
 // Top level schema //
-
-"evaluation-plans": [...#EvaluationPlan]
+#FrameworkEvaluation: {
+    // name of the framework being evaluated
+    name: string
+    // ID of the framework being evaluated
+    frameworkID: string
+    // one or more evaluations of the framework controls
+    evaluations: [#ControlEvaluation, ...]
+}
 
 // Types
 
+#URL: =~"^https?://[^\\s]+$"
+
 #ControlEvaluation: {
+    // name of the control being evaluated
     name: string
-    "control-id": string
-    result: #Result
-    message: string
-    "documentation-url"?: =~"^https?://[^\\s]+$"
-    "corrupted-state"?: bool
-    "assessment-results"?: [...#AssessmentResult]
+    // ID of the control being evaluated
+    controlID: string
+    // one or more assessments of the control requirements
+    assessments: [#Assessment, ...]
 }
 
-#AssessmentResult: {
-    result: #Result
+#Assessment: {
+    // ID of the requirement being evaluated
+    requirementID: string
+    // the methods used to assess the requirement
+    methods: [#AssessmentMethod, ...]
+}
+
+#AssessmentMethod: {
+    // the method used to assess the requirement
     name: string
+    // detailed explanation of the method
     description: string
-    message: string
-    "function-address" string
-    change?: #Change
-    value?: _
+    // did it run?
+    run: bool
+    // the result of the assessment, only present when run is true
+    //  * passed when all evidence suggests the control is met
+    //  * failed when some evidence suggests the control is not met
+    //  * needs_review when evidence was gathered but cannot be reliably interpreted to reach a decision. A human should review the evidence gathered
+    //  * error when the method/strategy failed to execute
+    result?: "passed" | "failed" | "needs_review | error"
+    remediation_guide?: #URL
 }
-
-#Result: "Passed" | "Failed" | "Needs Review"
-
-#Change: {
```

Comment:
There is suspected existing usage of this definition/type in a currently private repository that I do not have access to. Until I can see that usage and understand it better, this specific removal should be considered for discussion only.